secant/Info.plist

```diff
@@ -23,6 +23,17 @@
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
 	<string>$(MARKETING_VERSION)</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>zcash</string>
```

Comment:
According to [this documentation](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) this URL Type should be able to handle a URI like `zcash:{address}` could we verify this?